README.md

```diff
@@ -1,70 +1,111 @@
-
-Second, install npm packages:
-```bash
-npm install
-```
-*Note: This needs to be run when you initially setup the repository and when packages are updated/added/removed.*
-
-Third, to start the local server, use:
-```bash
-npm start
-```
-The app should then open in a browser window as `http://localhost:3000`
-
-### VS  Code Startup Configurations
-We can set launch configurations in VS Code for an improved development experience. This allows us to set breakpoints directly in the editor, and launch the app from the debug tab in VS Code. Multiple configurations can be added. Here is an example that launches the app with the Chrome browser:
+# Sero-can-webapp
+
+Code for [Serotracker.com](https://serotracker.com/).
+
+# Table of contents
+
+- [Sero-can-webapp](#sero-can-webapp)
+- [Table of contents](#table-of-contents)
+- [Set up](#set-up)
+  - [Installing Node & NPM](#installing-node--npm)
+    - [macOS/ Linux / WSL](#macos-linux--wsl)
+    - [Windows](#windows)
+  - [Local Development Setup](#local-development-setup)
+    - [Setting up the Application](#setting-up-the-application)
+    - [Running the Application Locally](#running-the-application-locally)
+      - [Alternative 1: Using the Terminal](#alternative-1-using-the-terminal)
+      - [Alternative 2: Using VS Code Configurations](#alternative-2-using-vs-code-configurations)
+- [Github Pages/Heroku Deployment](#github-pagesheroku-deployment)
+  - [Adding Secrets](#adding-secrets)
+    - [GitHub](#github)
+    - [Heroku](#heroku)
+  - [Staging Deployment](#staging-deployment)
+- [Continuous Integration](#continuous-integration)
+
+# Set up
+## Installing Node & NPM
+
+### macOS/ Linux / WSL
+We recommend using Node Version Manager ([nvm](https://github.com/nvm-sh/nvm)), which is a command line tool that allows you to quickly install, use, and switch between different versions of node. Using your terminal, follow these instructions:
+
+1. Install cURL (a tool used for downloading content from the internet in the command-line)
+
+- macOS (homebrew) `brew install  curl`
+- Ubuntu/WSL (apt) `sudo apt-get install curl`
+
+2. Install NVM. The following command will download version 0.39.1 of nvm. Check nvm's [official GitHub projects page](https://github.com/nvm-sh/nvm) for the latest release of NVM.
+
+- Bash: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`
+- Zsh: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | zsh`
+
+3. To verify your installation, run `nvm -v`. You may need to source your shell's config file using `source ~/.bash_profile` or `source ~/.zshrc`
+4. You can list which versions of Node are currently installed (should be none at this point) with `nvm ls`.
+5. Install version 12.16.0 of Node.js using `nvm install 12.16.0`. You can also install the LTS release using `nvm install --lts` and the current release using `nvm install node`.
```

Comment:
This is a bit confusing -- if we want them to use 12.16.0 shouldn't we just say in step 2 to download that version? And provide the curl links for downloading that version? Cause we get them to download nvm with one version, then do nvm ls, and then basically repeat it with 12.16.0

Reply:
Step 2 is to install version 0.39.1 of **nvm**, which we then use in step 5 to isntall version 12.16.0 of **Node.js**.